user-guide/docs/css/ds-docs.css

```diff
@@ -24,7 +24,7 @@
 /* To also support underlining headings in hovered card for `.h1` et cetera */
 /* https://github.com/DesignSafe-CI/DS-User-Guide/blob/df64c11/user-guide/docs/usecases/overview.md?plain=1#L34-L37 */
 /* https://github.com/TACC/Core-Styles/blob/v2.27.0/src/lib/_imports/components/c-card--docs.css#L23-L29 */
-[class*=card--]:hover :is( .h1, .h2, .h3, .h4, .h5, .h6 ) {
+a[class*=card--]:hover :is( .h1, .h2, .h3, .h4, .h5, .h6 ) {
```

Comment:
I fixed a style bug that only showed up when I change the cards to not be links.